compose.yaml

```
@@ -0,0 +1,265 @@
+      - API_GRAPHQL_OBJECT_FIELD_COST=${API_GRAPHQL_OBJECT_FIELD_COST:?error}
+      - API_GRAPHQL_LIST_FIELD_COST=${API_GRAPHQL_LIST_FIELD_COST:?error}
+      - API_GRAPHQL_NON_PAGINATED_LIST_FIELD_COST=${API_GRAPHQL_NON_PAGINATED_LIST_FIELD_COST:?error}
+      - API_GRAPHQL_MUTATION_BASE_COST=${API_GRAPHQL_MUTATION_BASE_COST:?error}
+      - API_GRAPHQL_SUBSCRIPTION_BASE_COST=${API_GRAPHQL_SUBSCRIPTION_BASE_COST:?error}
+      - API_RATE_LIMIT_BUCKET_CAPACITY=${API_RATE_LIMIT_BUCKET_CAPACITY:?error}
+      - API_RATE_LIMIT_REFILL_RATE=${API_RATE_LIMIT_REFILL_RATE:?error}
+      - CI=${CI:?error}
+      - NODE_ENV=${NODE_ENV:?error}
+    # https://docs.docker.com/reference/compose-file/build/#using-build-and-image
+    # https://docs.docker.com/reference/compose-file/build/#publishing-built-images
+    # https://docs.docker.com/reference/compose-file/services/#image 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)